wandb/cli/cli.py

```diff
@@ -1450,7 +1486,8 @@ def scheduler(
                 _args = int(_args)
             kwargs[_key] = _args
     try:
-        _scheduler = load_scheduler("sweep")(
+        sweep_type = kwargs.get("sweep_type", "wandb")
```

Comment:
"wandb" sweep, is this enough info? Should we indicate its the `wandb_server_scheduler`? I guess this key is set by users, so wandb might be best, if a little limiting